AudioPlayer-topic1/src/AudioPlayer.java

```diff
@@ -0,0 +1,21 @@
+public class AudioPlayer {

```

Comment:
Could you add a comment here letting me know which pattern did you choose to implement the Audio Player?